packages/edit-post/src/editor.native.js

```diff
@@ -195,12 +195,9 @@ export default compose( [
 		const { isFeatureActive, getEditorMode, getHiddenBlockTypes } =
 			select( editPostStore );
 		const { getBlockTypes } = select( blocksStore );
-		const { getDeviceType } = select( editorStore );
 
 		return {
-			hasFixedToolbar:
-				isFeatureActive( 'fixedToolbar' ) ||
-				getDeviceType() !== 'Desktop',
+			hasFixedToolbar: isFeatureActive( 'fixedToolbar' ),
```

Comment:
Don't we still need this? Although I see the floating toolbar even in trunk for the post editor...